awsconfigs/apps/katib-external-db-with-kubeflow/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+# The katib-external-db manifest does not create the necessary katib ui virtual service and doesn not apply the configurations from katib-cert-manager, which the katib-with-kubeflow manifest does. 
```

Comment:
nit: typo should be doesn't or does not

Reply:
@surajkota It's in the middle of the sentence, you can try a lookup on `virtual service and doesn not apply` to find it